test/TestIncludeMissingFileRule.py

```diff
@@ -0,0 +1,85 @@
+import os
+import pytest
+
+from ansiblelint import Runner, RulesCollection
+
+
+class PlayFile:
+    def __init__(self, name, content):
+        self.name = name
+        self.content = content
```

Comment:
or you could just use a namedtuple instead of creating a class manually
```suggestion
from collections import namedtuple
import os

import pytest

from ansiblelint import Runner, RulesCollection


PlayFile = namedtuple('PlayFile', ['name', 'content'])
```